deeplinkdispatch-processor/src/main/java/com/airbnb/deeplinkdispatch/DeepLinkProcessor.java

```diff
@@ -517,6 +525,7 @@ public ParameterSpec apply(TypeElement typeElement) {
       .superclass(CLASS_BASE_DEEP_LINK_DELEGATE)
       .addMethod(constructor)
       .addMethod(constructorWithPathVariables)
+      .addOriginatingElement(originatingElement)
```

Comment:
I don't thin it matters since the registry classes are generated.. but it might be most accurate to mark them as originating elements as well.